src/react-apps/applications/runtime/src/features/form/containers/WorkflowStep.tsx

```diff
@@ -217,7 +218,10 @@ class WorkflowStepComponent extends React.Component<IWorkflowStepProps, IWorkflo
     return (
       <div id='workflowContainer' style={{ backgroundColor, height: 'calc(100vh - 146px)' }} >
         <div className='container'>
-          {this.renderTop()}
+          <AltinnAppHeader
```

Comment:
If the renderTop() method is not used anymore, it can be removed.

Reply:
Removed.